docs/source/plugin/bot.rst

```diff
@@ -335,3 +335,135 @@ You can also iterate over the list of users and filter them by privileges::
 
     Read about the :class:`~sopel.tools.target.Channel` and
     :class:`~sopel.tools.target.User` classes for more details.
+
+
+About time
+==========
+
+Your plugin may want to display dates and times in messages. For that, you can
+always count on the :mod:`datetime` built-in module. However, what if you would
+like to respect the date-format for a given user or a given channel?
+:mod:`Sopel<sopel.tools.time>` can help you with that with various functions:
+
+* :func:`~sopel.tools.time.get_timezone` will fetch the right timezone for you
+* :func:`~sopel.tools.time.format_time` will format your aware datetime for you
+
+Here is a full example of that::
+
+    import datetime
+
+    import pytz
```

Comment:
```suggestion
    import pytz  # always available; Sopel requires it
```

LMK if you think it's _more_ confusing to call attention to this.

Reply:
I'd rather put a note at the end than in the code sample.

Reply:
That's fair.